website/source/docs/configuration/interpolation.html.md

```diff
@@ -195,6 +195,12 @@ The supported built-in functions are:
       Example: `element(split(",", var.r53_failover_policy), signum(count.index))`
       where the 0th index points to `PRIMARY` and 1st to `FAILOVER`
 
+  * `sort(list)` - Returns a lexographically sorted list of the strings contained in
+      the list passed as an argument. Sort may only be used with lists which contain only
+      strings.
+      Example: `sort(aws_instance.foo.*.id)`
+      Example: `sort(var.list_of_strings)`
```

Comment:
I think the newlines won't print well on the site - might want to just merge these up inline like `Examples: `foo`, `bar``